jenkins/runners/systemd-pr-build-vagrant-sanitizers.sh

```diff
@@ -19,6 +20,17 @@ ARGS=
 
 if [ "$ghprbPullId" ]; then
     ARGS="$ARGS --pr $ghprbPullId "
+
+    # We're not testing the master branch, so let's see if the PR scope
+    # is something we should indeed test
+    git clone https://github.com/systemd/systemd systemd-tmp && cd systemd-tmp
+    git fetch -fu origin "refs/pull/$ghprbPullId/head:pr"
+    git checkout pr
+    git rebase master
+    if ! git diff master --name-only | grep -E '(^(catalog|factory|hwdb|meson*|network|[^\.].*.d|rules|src|test|units))'; then
+        echo "Changes in this PR don't seem relevant, skipping..."
+        exit 0
+    fi
```

Comment:
Is the rebase necessary? Maybe just do `git diff $(git merge-base master) instead`?

Reply:
Ooh, this is neat, thank you, didn't know about this!